### PR TITLE
[Feat] Add -y option to Aim CLI commands

### LINKED ISSUE #2440: Introduce `-y` option in aim CLI commands
## 🚀 Feature

Aim commands such as `aim init` prompt user for inputs, which are mainly confirmation prompts. Need to add a command flag that will automatically confirm the action.

### Motivation
Enable aim commands in non-interactive environments (shell scripts, etc.)

### Pitch
Add -y option similar to apt install to automatically confirm the action.

### PR DESCRIPTION
**Pull request by emekaokoli19**
Resolves #2440 @alberttorosyan or @gorarakelyan please take a look at this

### CODE REVIEW

**Comment by gorarakelyan** [comment]
@emekaokoli19 thanks for the contribution! We will review it shortly and try to ship with the upcoming 3.16.3 version (planned for this week).

File: aim/cli/up/commands.py [hunk 1]
```diff
@@ -75,7 +76,10 @@ def up(dev, host, port, workers, uds,
     if repo_status == RepoStatus.MISSING:
         init_repo = None
         if not force_init:
```
Comment (alberttorosyan) [inline aim/cli/up/commands.py:78]: As we have a dedicated `force_init` flag, does it make sense to check the `--yes` here?

**Review by alberttorosyan — COMMENTED** [review]
@emekaokoli19 great work! Thanks for your contribution. Left a comment related to the `aim up` command.

File: aim/cli/up/commands.py (hunk 1 above)
Comment (emekaokoli19) [reply aim/cli/up/commands.py:78]: @alberttorosyan sorry for the late reply, I didn't notice the `force_init` flag, will add a commit to fix this now

**Pull request closed without merging**